modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -59,6 +72,58 @@ If you provision the adapter with organization credentials, you will be able to
 If you specify environment credentials, you will only be able to manage APIs of that environment. MuleSoft recommends that you use the environment credentials instead of the organization credentials. For more information about which permissions to use, see xref:api-manager::org-credentials-config-mule4.adoc#obtaining-credentials[obtaining credentials].
 * Cluster-admin role access to the Kubernetes cluster.
 
+== Configuring Your Environment Variable
+
+Before you begin installing Anypoint Service Mesh, update your PATH environment variable to store system-wide values, such as the directories to search for the executable programs.
```

Comment:
PATH > `PATH` , to avoid translation issues.

Reply:
Done! Thanks!